src/site/content/en/learn/pwa/windows/index.md

```diff
@@ -145,7 +145,12 @@ To reduce phishing within PWAs, some desktop browsers may have additional measur
 
 An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see in the following image:
 
-<video autoplay="" loop="" muted=""><source src="https://storage.googleapis.com/web-dev-uploads/video/8WbTDNrhLsU0El80frMBGE4eMCD3/JwN0btyXFGiT9oPFh2qJ.webm" type="video/webm"></video>
+{% Video src="video/W3z1f5ZkBJSgL1V1IfloTIctbIF3/I0idt5S5IvAu5iy0WCq7.mp4",
+autoplay="true",
+loop="true",
+muted="true"
+%}
```

Comment:
Tested this and it STILL doesn't work :-(

Apparently [iOS won't allow autoplaying](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030251) unless you also specify `playsinline`.

Probably want to add controls to allow it to stop playing while you're at it. But annoyingly that shows the controls by default which is a little ugly. So next best option I've got is to not autoplay but instead add a poster image:

```suggestion
loop="true",
muted="true",
playsinline="true",
controls="true",
poster="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/emg7quu4DyvYwlyzgDE1.png"
%}
```